src/app.scss

```diff
@@ -37,23 +37,23 @@ body {
 	/* TODO properly name these */
 	--main-font: 'Manrope', sans-serif;
 	--mono-font: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
-	--white: hsl(206, 100%, 94%);
-	--accent-color: hsl(206, 100%, 81%);
-	--accent-color-two: hsl(208, 75%, 82%);
-	--accent-low-opacity: hsla(205, 91%, 69%, 0.15);
-	--bg-color: hsl(252, 10%, 11%);
-	--grey-one: hsl(210, 14%, 17%);
-	--grey-two: hsl(212, 19%, 19%);
-	--grey-three: hsl(221, 17%, 26%);
-	--grey-four: hsl(226, 48%, 18%);
-	--grey-five: hsl(208, 30%, 75%);
-	--grey-six: hsl(230, 9%, 13%);
-	--grey-seven: hsl(240, 9%, 13.5%);
-	--grey-eight: hsla(207, 30%, 75%, 0.577);
-	--grey-nine: hsla(240, 6%, 7%, 0.3);
-	--grey-ten: hsl(230, 9.5%, 17.5%);
-	--grey-eleven: hsl(208, 10%, 40%);
-	--red-one: hsl(333, 84%, 62%);
+	--white: hsl(calc(var(--hue, 206) + 0), 100%, 94%);
+	--accent-color: hsl(calc(var(--hue, 206) + 0), 100%, 81%);
+	--accent-color-two: hsl(calc(var(--hue, 206) + 2), 75%, 82%);
+	--accent-low-opacity: hsla(calc(var(--hue, 206) - 1), 91%, 69%, 0.15);
+	--bg-color: hsl(calc(var(--hue, 206) + 46), 10%, 11%);
+	--grey-one: hsl(calc(var(--hue, 206) + 4), 14%, 17%);
+	--grey-two: hsl(calc(var(--hue, 206) + 6), 19%, 19%);
+	--grey-three: hsl(calc(var(--hue, 206) + 15), 17%, 26%);
+	--grey-four: hsl(calc(var(--hue, 206) + 20), 48%, 18%);
+	--grey-five: hsl(calc(var(--hue, 206) + 2), 30%, 75%);
+	--grey-six: hsl(calc(var(--hue, 206) + 24), 9%, 13%);
+	--grey-seven: hsl(calc(var(--hue, 206) + 34), 9%, 13.5%);
+	--grey-eight: hsla(calc(var(--hue, 206) + 1), 30%, 75%, 0.577);
+	--grey-nine: hsla(calc(var(--hue, 206) + 34), 6%, 7%, 0.3);
+	--grey-ten: hsl(calc(var(--hue, 206) + 24), 9.5%, 17.5%);
+	--grey-eleven: hsl(calc(var(--hue, 206) + 2), 10%, 40%);
+	--red-one: hsl(calc(var(--hue, 206) + 127), 84%, 62%);
```

Comment:
i would say "red is semantic, red means danger" but apparently we only use on for the donate page? idk, it's your choice if you want to make red constant or dynamic

Reply:
@oSumAtrIX what's your thoughts

Reply:
I do not understand what happens here, whats the question?

Reply:
should the var --red-one keep the original hue or being shifted?

Reply:
Why are we shifting hues

Reply:
https://github.com/ReVanced/revanced-website/pull/227#discussion_r1499642273

Reply:
Can you elaborate what you mean?

Reply:
@KTibow

Reply:
@oSumAtrIX 

Before we were hardcoding colors that we switched to for each event
Now we're hardcoding hue offsets - eg by default we have a 206 hue offset, but on easter we have a 100 hue offset. This affects most of the colors and makes them match with the scheme.
The question is whether red should always be red or change on the special events.

Reply:
The variable red should always be red but I can imagine it being difficult for this PR to change every reference from red to some other variable. Any suggestions?

Reply:
resolved in 2fd62af